src/app/components/home/home.component.html

```diff
@@ -17,9 +17,9 @@
   </mat-grid-tile>
   <mat-grid-tile>
     <span>
-      <span>Runs on <a mat-button onclick="window.open('//http://firebase.google.com/')"><b>Firebase</b></a>, made with <a mat-button onclick="window.open('//angular.io')"><b>Angular</b></a></span>
+      <span>Runs on <a mat-button onclick="window.open('https://firebase.google.com/')"><b>Firebase</b></a>, made with <a mat-button onclick="window.open('//angular.io')"><b>Angular</b></a></span>
       <br/>
-      Charts powered by <a mat-button onclick="window.open('//amcharts.com?ref=quantified-self.io')"><b>amCharts</b></a>
+      Charts powered by <a mat-button onclick="window.open('https://amcharts.com?ref=quantified-self.io')"><b>amCharts</b></a>
```

Comment:
This one is ok without HTTPS. 
When you ommit the protocol eg `//facebook.com` the current app protocol will be used. 
But that means that if someone would run the app on non HTTPS it would redirect to non HTTPS that is not good. So your solution works and fixes this. 

Thanks